src/test/java/io/confluent/connect/jdbc/source/integration/BaseConnectorIT.java

```diff
@@ -0,0 +1,228 @@
+    }
+  }
+
+  protected void addConnectionProperties(Map<String, String> props) {
+    props.put("connection.password", "password");
+    props.put("connection.user", "user");
+    props.put("connection.url", "jdbc:mysql://localhost:3307/db");
+    props.put("dialect.name", "MySqlDatabaseDialect");
+    props.put("value.converter", JsonConverter.class.getName());
+    props.put("topic.prefix",  "sql-");
+  }
+
+  protected void dropTableIfExists(String tableName) throws SQLException {
+    Statement st = connection.createStatement();
+    st.executeQuery("drop table if exists " + tableName);
+  }
+
+  protected void createTable() throws SQLException {
+    Statement st = connection.createStatement();
+    String sql = "CREATE TABLE " + KAFKA_TOPIC +
+            "(id INTEGER not NULL, " +
+            " first_name VARCHAR(255), " +
+            " last_name VARCHAR(255), " +
+            " age INTEGER, " +
+            " PRIMARY KEY ( id ))";
+    st.executeQuery(sql);
+  }
+
+  protected void createTimestampTable() throws SQLException {
+    Statement st = connection.createStatement();
+    String sql = "CREATE TABLE " + KAFKA_TOPIC +
+            "(id int not NULL, " +
+            " first_name VARCHAR(255), " +
+            " last_name VARCHAR(255), " +
+            " time1 TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP );";
+    st.executeQuery(sql);
+  }
+
+  protected void sendTestTimestampDataToMysql(int start, int numOfRecords) throws SQLException {
+    String sql = "INSERT INTO testtable(id,first_name,last_name) "
+            + "VALUES(?,?,?)";
+    PreparedStatement pstmt = connection.prepareStatement(sql);
+    for (int i=start; i<start+numOfRecords; i++) {
+      pstmt.setInt(1, i);
+      pstmt.setString(2, "FirstName");
+      pstmt.setString(3, "LastName");
+      pstmt.executeUpdate();
+    }
+    pstmt.close();
+  }
+
+  protected void sendTestDataToMysql(int start, int numOfRecords) throws SQLException {
+    String sql = "INSERT INTO testtable(id,first_name,last_name,age) "
+            + "VALUES(?,?,?,?)";
+    PreparedStatement pstmt = connection.prepareStatement(sql);
+    for (int i=start; i<start+numOfRecords; i++) {
+      pstmt.setLong(1, i);
+      pstmt.setString(2, "FirstName");
+      pstmt.setString(3, "LastName");
+      pstmt.setLong(4, 20);
+      pstmt.executeUpdate();
+    }
+    pstmt.close();
+  }
+
+  protected void restartConnectorTask(int taskID) {
+    // Restart the task having specific 'taskID'.
+    String url = connect.endpointForResource(String.format("connectors/%s/tasks", CONNECTOR_NAME));
+    url = url + String.format("/%s/restart", String.valueOf(taskID));
+    CloseableHttpClient httpClient = HttpClients.createDefault();
+    HttpPost postRequest = new HttpPost(url);
+    postRequest.addHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
+    try {
+      httpClient.execute(postRequest);
+    } catch (IOException e) {
+      throw new RuntimeException("Unable to restart the task");
```

Comment:
```suggestion
      throw new RuntimeException("Unable to restart the task", e);
```
Don't swallow the exception except to avoid duplicate trace in logs.